pkg/util/util.go

```diff
@@ -35,3 +36,7 @@ func CopyStrings(s []string) []string {
 func GetUUID() string {
 	return string(uuid.NewUUID())
 }
+
+func ReadObject(o *v1alpha1.ObjectReference) string {
```

Comment:
Since, we're just using a stub for now, maybe add a TBD comment above this function?

Reply:
fixed